### DISCUSSION TITLE
Support scattered progress bar

### DISCUSSION THREAD

**Discussion by techeverri** [trakt/showly#169] [Ideas / Feature Requests]
Trakt supports a progress bar that shows spaces (holes) organized chronologically for the unwatched episodes.
Example screenshots below 👇 
_Note: don't asked me why I skipped an entire season, but it happened_ 🙈 

## Trakt

<table>
    <tr>
        <td>
            <img src="https://user-images.githubusercontent.com/3720424/101553030-3a4db600-39b4-11eb-8ca9-e8785da5a21c.png">
        </td>
        <td>
            <img src="https://user-images.githubusercontent.com/3720424/101553085-58b3b180-39b4-11eb-8bbf-569d6ddd541d.png">
        </td>
        <td>
            <img src="https://user-images.githubusercontent.com/3720424/101553209-91ec2180-39b4-11eb-93d0-282c5b26e07b.png">
        </td>
    </tr>
</table>

## Showly

<table>
    <tr>
        <td>
            <img src="https://user-images.githubusercontent.com/3720424/101553840-c1e7f480-39b5-11eb-8d63-61eb465e3f28.png">
        </td>
        <td>
            <img src="https://user-images.githubusercontent.com/3720424/101553763-96650a00-39b5-11eb-9ffe-e7b5c337c2f4.jpg">
        </td>
    </tr>
</table>

**Comment by michaldrabik** [comment]
Yeah I think this can be done and wouldn't take that much time. Just need to create some custom progress bar component.

#171
